ui/src/App.tsx

```diff
@@ -1,52 +1,81 @@
 //External components
 import { ThemeProvider } from "@mui/material/styles";
-import { Container } from "@mui/material";
+import { Container, Alert } from "@mui/material";
 
 //Internal components
 import TopBar from "./components/TopBar/TopBar";
 import ImportScreen from "./ImportScreen";
 import ValidatorList from "./components/ValidatorList/ValidatorList";
-import Message from "./components/Messages/Message";
-
-//Styles
-import "./App.css";
+import { consensusClient, executionClient } from "./params";
 
 //Themes
 import { darkTheme } from "./Themes/globalThemes";
 
 //Logic
-import { getUrlParams } from "./getUrlParams";
+import { getUrlParams } from "./logic/Utils/getUrlParams";
 import { Web3SignerApi } from "./apis/web3signerApi";
 
 //Other libraries
 import { BrowserRouter, Route, Routes } from "react-router-dom";
 import React, { useEffect } from "react";
+import ClientsBox from "./components/ClientsBox/ClientsBox";
 
 function App() {
   const [network, setNetwork] = React.useState("");
   const [web3signerApi, setWeb3signerApi] =
     React.useState<Web3SignerApi | null>(null);
+  const [signerStatus, setSignerStatus] = React.useState("loading");
 
   useEffect(() => {
-    const { network, authToken, host, baseUrl } = getUrlParams();
+    const { network, authToken, host, signerUrl } = getUrlParams();
```

Comment:
I'll try to clarify it and improve it, but right now URL params have priority above global envs, because I think it is a good way of changing the params easily if you want to. What do you think about this?

Reply:
https://github.com/dappnode/eth2-keymanager-frontend/commit/e61e492fe9beb6480244c43cde8ed4f339ca16d1. Check this carefully because I am not sure if it fits with your comment.

Reply:
https://github.com/dappnode/eth2-keymanager-frontend/commit/a7856ca2181de324c2fe84cc4d4c7ce9f516311b